src/Symfony/Component/Console/Command/Command.php

```diff
@@ -429,9 +429,9 @@ public function addArgument(string $name, int $mode = null, string $description
     /**
      * Adds an option.
      *
-     * @param $shortcut The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
-     * @param $mode     The option mode: One of the InputOption::VALUE_* constants
-     * @param $default  The default value (must be null for InputOption::VALUE_NONE)
+     * @param string|array|null         $shortcut The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
```

Comment:
should be `string|string[]|null` in phpdoc, to also document the type of array items